models/balancer/ethereum/balancer_v2_ethereum_pools_fees.sql

```diff
@@ -0,0 +1,42 @@
+{{
+    config(
+        schema = 'balancer_v2_ethereum',
+        alias='pools_fees',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_number', 'tx_hash', 'index'],
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "balancer_v2",
+                                    \'["masquot", "jacektrocinski"]\') }}'
```

Comment:
Very small detail but masquot didn't contribute to this view, he moved all views to the deprecated folder and that's why he has the last commit. If you browse further the history, you'll find my PR https://github.com/duneanalytics/spellbook/pull/497

Reply:
You deserve credit!

I updated the code and added your Dune username, `metacrypto`, instead of `masquot`.